src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -106,11 +108,29 @@ public void configure() {
     driveConfiguration.Slot0.kI = prefDrivetrain.driveI.getValue();
     driveConfiguration.Slot0.kD = prefDrivetrain.driveD.getValue();
 
+    driveConfiguration.CurrentLimits.SupplyCurrentLimitEnable = true;
+    driveConfiguration.CurrentLimits.SupplyCurrentThreshold = 50;
+    driveConfiguration.CurrentLimits.SupplyCurrentLimit = 40;
+    driveConfiguration.CurrentLimits.SupplyCurrentThreshold = 0.1;
```

Comment:
dude how did our drivetrain even drive what the

Reply:
I was gonna make these into preferences before making a pr but this has set me back

Reply:
as it stands, current limiting is being applied for any time that the motor draws more than 50 amps.
"Delay supply current limiting until current exceeds this threshold for longer than SupplyTimeThreshold.  This allows current draws above SupplyCurrentLimit for a fixed period of time.  This has no effect if SupplyCurrentLimit is greater than this value."